src/tribol/geom/GeomUtilities.cpp

```diff
@@ -1137,10 +1138,9 @@ bool SegmentIntersection2D( const real xA1, const real yA1, const real xB1, cons
      yDiff = (yDiff < 0.) ? -1.0 * yDiff : yDiff;
 
      real diffTol = 1.0E-3;
-     SLIC_ERROR_IF( xDiff > diffTol || yDiff > diffTol, 
+     SLIC_DEBUG_IF( xDiff > diffTol || yDiff > diffTol, 
```

Comment:
This will be part of a future PR refining how the logging is done. There are a few spots where DEBUG prints are contingent on debug calculations. We need a solid and uniform way to guard these. (future PR)